platform/android/os_android.cpp

```diff
@@ -223,10 +222,7 @@ bool OS_Android::request_permission(const String &p_name) {
 
 Error OS_Android::open_dynamic_library(const String p_path, void *&p_library_handle, bool p_also_set_library_path) {
 	p_library_handle = dlopen(p_path.utf8().get_data(), RTLD_NOW);
-	if (!p_library_handle) {
-		ERR_EXPLAIN("Can't open dynamic library: " + p_path + ". Error: " + dlerror());
-		ERR_FAIL_V(ERR_CANT_OPEN);
-	}
+	ERR_FAIL_COND_V_MSG(!p_library_handle, ERR_CANT_OPEN, "Can't open dynamic library: " + p_path + ", Error: " + dlerror() + ".");
```

Comment:
The message was more correct before, either "Error" should be uncapitalized, or if should be preceded by a dot and not a comma.

Reply:
Since I think I saw it in other code, I'll go with a comma as it is but then uncapitalized instead.